crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -455,14 +451,8 @@ impl<M: NetworkMsg, K: SignatureKey + 'static> Libp2pNetwork<M, K> {
         async_spawn({
             let is_ready = self.inner.is_ready.clone();
             async move {
-                let bs_addrs = loop {
-                    let bss = bootstrap_ref.read().await;
-                    let bs_addrs = bss.clone();
-                    drop(bss);
-                    if bs_addrs.len() >= num_bootstrap {
-                        break bs_addrs;
-                    }
-                };
```

Comment:
Yeah, as nodes spun up they would add their own `PeerId` and `Multiaddr` to the same, shared RwLock, then we had this spinlock to check the number.

Since I'm going to be looking at bootstrap soon anyway, I figured the easiest solution was to just take the current value. This comes with the consequence of the first node not having any peers initially until other nodes start up and connect to it, which is fine IMO